tracr/compiler/craft_graph_to_model.py

```diff
@@ -27,38 +27,27 @@
 NodeID = nodes.NodeID
 
 
-def _get_longest_path_length_to_node(graph: nx.DiGraph, sources: Sequence[Node],
-                                     node: Node) -> int:
-  """Returns the lengths of the longest path from sources to node.
-
-  Only SOps count towards the length of a path.
-
-  Args:
-    graph: DAG to compute longest path in.
-    sources: List of starting nodes, longest path will be a maximum over all.
-    node: Target node.
-
-  Returns:
-    Number of steps needed for the longest path from the source to the node, or
-    -1 if there is no path from any of the sources to the target node.
-  """
-  if node in sources:
-    return 0
-
-  def num_sops(path: Sequence[NodeID]) -> int:
-    num = 0
-    for node_id in path:
-      if isinstance(graph.nodes[node_id][nodes.EXPR], rasp.SOp):
-        num += 1
-    return num
+def compute_computational_depth(graph: nx.DiGraph, sources: Node):
+  """returns the computational depth of each node in the graph,
```

Comment:
Docstring should have a single line summary (see https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings). Please generally follow the Google Python style guide.